chart/kubeapps/templates/kubeappsapis/deployment.yaml

```diff
@@ -166,6 +166,8 @@ spec:
                 secretKeyRef:
                   key: postgres-password
                   name: {{ include "kubeapps.postgresql.secretName" . }}
+            - name: OCI_CATALOG_URL
+              value: {{ printf ":%d" (int .Values.ociCatalog.containerPorts.http) | quote }}
```

Comment:
Haven't checked, but I assume `containerPorts.http` is actually pointing to the grpc endpoint, no?

Reply:
Yes, there is only one endpoint. Perhaps we should name it grpc instead... (I think the container template I used to create the ocicatalog helm section used http).

Reply:
Done.

Reply:
Thanks! Much more descriptive this way